use std::error::Error;

pub fn test_kattis() -> Result<(), Box<dyn Error>> {
    println!("YOU ARE TESTING ME");
    Ok(())
}
